.github/workflows/e2e-nightly-master.yml

```diff
@@ -6,6 +6,13 @@
 name: e2e-nightly-master
 on:
   workflow_dispatch:  # allow running workflow manually
+    inputs:
```

Comment:
I think maybe the indentation isn’t quite right here. Are the inputs really specific to the trigger?

Reply:
I coped it from: https://github.com/tendermint/tendermint/blob/4ba5a053dea17f60dfbf0b68f6f477bbfc598b85/.github/workflows/jepsen.yml#L13

Reply:
TIL